pkg/promlib/querydata/request.go

```diff
@@ -5,21 +5,21 @@ import (
 	"fmt"
 	"net/http"
 	"regexp"
+	"sync"
 	"time"
 
+	"github.com/grafana/dskit/concurrency"
```

Comment:
since we are exporting prometheus backend as a module `promlib` (for example: https://github.com/grafana/grafana/releases/tag/pkg%2Fpromlib%2Fv0.0.6) we need to import this from a general place. I'll talk to some people to move this package  `concurrency` in a general place. Maybe we will consider moving it in https://github.com/grafana/grafana-plugin-sdk-go

Reply:
Oh 🤦 , It is already in a general place. My bad. So you can ignore the comment above